### PR TITLE
Math Magicians App: React Testing Library 

### PR DESCRIPTION
**Pull request by Richie9000**
# Documentation

All the documentation is on the README file.

## Updates:

* Added React Testing Library.
* Run tests using snapshot.
* Run test for calculate.js and operate.js

## Authors:
@richie9000
@smunozmo 

## Notes:

✅ - Used React Testing Library.
✅ - Used React.
✅ - Used React router.
✅ - No linters errors.

### CODE REVIEW

File: src/tests/calculate.test.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import calculate from '../components/logic/calculate';
```
Comment (ZawadiSumba66) [inline src/tests/calculate.test.js:1]: - [x] There ought to be the usage of Jest snapshots for testing all the components i.e files in the components and pages folders. Please add them.

**Review by ZawadiSumba66 — CHANGES_REQUESTED** [review]
#### Hi @Richie9000 👋 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._
-  [ ] Please include ` npm run test ` to run the tests.
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/tests/calculate.test.js (hunk 1 above)
Comment (Dipeshtwis) [reply src/tests/calculate.test.js:1]: Well Addressed !!!

**Review by Dipeshtwis — APPROVED** [review]
# Status: Approved ✔️

## Hi @smunozmo &  @Richie9000, 

Good job on implementing the suggested changes. Well Done.
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

**You may close this Pull Request and merge the changes**

  
**Keep being awesome devs and good luck with your next milestone!**
---------

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
_**Dipesh Kumar**_

--------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by smunozmo** [comment]
Thank you @Dipeshtwis @ZawadiSumba66

**Pull request merged**